src/local/butler/deploy.py

```diff
@@ -525,11 +542,18 @@ def execute(args):
     deploy_k8s = False
     deploy_zips = True
 
+  release = args.release
+  if args.release == 'candidate':
```

Comment:
can we just use `args.release` directly? `argparse` validates the choices for us right?

Reply:
Good catch. It does
```
clusterfuzzvitorguidi@vitorguidi:~/projects/clusterfuzz$ python3.11 butler.py package --platform=linux --release=loremipsum
error: argument -r/--release: invalid choice: 'loremipsum' (choose from 'prod', 'candidate')
usage: butler.py package [-h] [-p {linux,macos,windows,all}] [-r {prod,candidate}]

options:
  -h, --help            show this help message and exit
  -p {linux,macos,windows,all}, --platform {linux,macos,windows,all}
  -r {prod,candidate}, --release {prod,candidate}
  ```